docs/mainnet-examples/ArbitrumTimelock.md

```diff
@@ -0,0 +1,219 @@
+# Arbitrum Timelock Proposal
```

Comment:
Instead of copying the entire proposal file and displaying it at once, I think we should write this in a tutorial format, writing one function at a time and explaining what the function is doing. This serves both for mainnet examples and guides.